tests/e2e/instrumentation-java/00-install-collector.yaml

```diff
@@ -4,8 +4,6 @@ metadata:
   name: sidecar
 spec:
   mode: sidecar
-  args:
```

Comment:
@pavolloffay, @anuraaga, could you please confirm this shouldn't affect the test?

Reply:
I would suggest reverting this change if it is not related to 0.43 release.

In all instrumentation tests the `args` does not matter and it was just copied over from other smoke tests. However it should stay in place in smoke-sidecar tests - it was added in this PR https://github.com/open-telemetry/opentelemetry-operator/pull/314/files#diff-67da78435bb33bdde063a374611b00510f546387ad4b9f8a50197649c3af0aabR7

Reply:
So the metrics args were removed in 0.43 https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.43.0.

so I would suggest fixing the smoke-sidecar to use a different argument to keep validating that args in the CR works.

Reply:
Ok, then I'll keep changes and open an issue to create/change the sidecar tests with different args.

Reply:
please create an issue and add it here to cross-reference it.

Reply:
https://github.com/open-telemetry/opentelemetry-operator/issues/702